DevCycle.SDK.Server.Local/ConfigManager/EnvironmentConfigManager.cs

```diff
@@ -205,6 +205,9 @@ private async Task FetchConfigAsyncWithTask()
 
                 logger.LogError(finalError.ErrorResponse.Message);
                 dvcEventArgs.Errors.Add(finalError);
+            } catch (Exception e) {
```

Comment:
is there no more specific exception you can be catching here? this basically catches any error at all

Reply:
yep i can use ArgumentNullException